tests/core/policies/test_unexpected_intent_policy.py

```diff
@@ -82,9 +87,16 @@ def featurizer(self) -> TrackerFeaturizer:
         return featurizer
 
     @staticmethod
-    def persist_and_load_policy(trained_policy: UnexpecTEDIntentPolicy, tmp_path: Path):
-        trained_policy.persist(tmp_path)
-        return UnexpecTEDIntentPolicy.load(tmp_path)
+    def persist_and_load_policy(
+        trained_policy: UnexpecTEDIntentPolicy,
+        model_storage: ModelStorage,
+        resource: Resource,
+        execution_context: ExecutionContext,
+    ):
+        trained_policy.persist()
```

Comment:
this is no longer necessary as the policy is automatically persisted as part of the training 
```suggestion
```